package.json

```diff
@@ -191,9 +192,6 @@
     "build:js": "NODE_ENV=production webpack --config webpack.config.cjs",
     "predev": "rm -rf assets/css/* assets/js/*",
     "dev": "webpack --config webpack.config.cjs --watch",
-    "env:start": "./bin/local-env/start.sh",
-    "env:stop": "./bin/local-env/stop.sh",
-    "env:reset-site": "./bin/local-env/install-wordpress.sh --reset-site",
```

Comment:
Any chance we could just make these aliases? Means I don't have to change what I try and no learning curve for current devs on the project.

Reply:
Yeah maybe we can keep them for some time.